docs/tutorials/prefect.md

```diff
@@ -0,0 +1,90 @@
+# Integrating nnbench with Prefect
+
+If you have more complex workflows it is sensible to use a workflow orchestration tools. 
+Benchmarking with nnbench can be integrated with orchestrators. We will present an example integration with Prefect.
+We will explain the orchestration concepts in a high level and link to the corresponding parts of the  Prefect [docs](https://docs.prefect.io/).
+The full example code is in the nnbench [repository](https://github.com/aai-institute/nnbench/tree/main/examples/prefec).
+
+In this example we want to orchestrate the training and benchmarking of a linear regression model.
+
+## Project Structure
+### Defining the training tasks and workflows
+We recommend to separate the training and benchmarking logic. 
+
+```python
+--8<-- "examples/prefect/training.py"
+```
+
+The `training.py` file contains functions to generate synthetic data for our regression model, facilitate a train-test-split, and finally train the regression model.
+We have applied Prefects [`@task` decorator.](https://docs.prefect.io/latest/concepts/tasks/)  
```

Comment:
```suggestion
We have applied Prefect's [`@task` decorator.](https://docs.prefect.io/latest/concepts/tasks/)  
```

I would also merge the next sentence into it, e.g. like "which marks the function logic ..."